src/main/kotlin/io/github/karmasmp/karma/player/creator/Creator.kt

```diff
@@ -0,0 +1,30 @@
+package io.github.karmasmp.karma.player.creator
+
+import io.github.karmasmp.karma.player.visuals.PlayerVisuals
+
+import org.bukkit.entity.Player
+
+import java.util.*
+
+object Creator {
```

Comment:
Would it not make more sense to just have an isLive boolean on the KarmaPlayer rather than keep an entirely separate object to store people who are live?

Reply:
Will tackle at a later date.